xwiki-platform-core/xwiki-platform-rendering/xwiki-platform-rendering-macros/xwiki-platform-rendering-macro-include/src/test/java/org/xwiki/rendering/internal/macro/IncludeMacroParametersTest.java

```diff
@@ -41,4 +41,14 @@ public void setPage()
 
         assertSame(EntityType.PAGE, parameters.getType());
     }
+    
+    @Test
+    public void setExcludeFirstHeading()
+    {
+        IncludeMacroParameters parameters = new IncludeMacroParameters();
+
+        parameters.setExcludeFirstHeading(true);
+        
+        assertSame(true, parameters.excludeFirstHeading());
```

Comment:
That's not best practice. We don't care if it's the same object.... This is just a boolean, we need to test for equality. Also you should use assertTrue...

Reply:
Got it. Thanks :)